src/scheduled_tasks.rs

```diff
@@ -58,6 +63,12 @@ pub fn setup(db_url: String, user_agent: String) -> Result<(), LemmyError> {
     clear_old_activities(&mut conn_3);
   });
 
+  // Remove old rate limit buckets every week
+  scheduler.every(CTimeUnits::weeks(1)).run(move || {
+    let week = Duration::from_secs(3600 * 24 * 7);
```

Comment:
Is there a particular reason for using one week? Afaik the rate limits all have default periods of one hour or so. Of course admins can change that, but clearing old entries after a day seems more than enough time.

Reply:
I now changed it to 1 hour.

Originally I set it to 1 week because I was concerned about the overhead of frequently calling `remove_older_than` and re-inserting buckets that were deleted. Now I realize that this can use much more memory than only storing very recent buckets, because different users visit the site at different times of day.